website/docs/developer-docs/setup/full-dev-environment.md

```diff
@@ -43,17 +92,19 @@ Generally speaking, authentik is a Django application, ran by gunicorn, proxied
 
 Most functions and classes have type-hints and docstrings, so it is recommended to install a Python Type-checking Extension in your IDE to navigate around the code.
 
-Before committing code, run the following commands in the same directory as your local authentik git repository:
-
-```shell
-make lint # Ensures your code is well-formatted
-make gen # Generates an updated OpenAPI Docs for any changes you make
-```
-
 ## Frontend Setup
 
 By default, no compiled bundle of the frontend is included so this step is required even if you're not developing for the UI.
 
+The UI requires the authentik API files for Typescript be built and installed:
+
+```
+$ make migrate # On a fresh install, ensures the API schema file is available
+$ make gen # Generates the API based on the schema file
+```
+
+If you make changes to the authentik API, you must re-run `make gen` so that the corresponding changes are made to API library used by the UI.
```

Comment:
```suggestion
If you make changes to the authentik API, you must re-run `make gen` so that the corresponding changes are made to the API library that is used by the UI.
```

Reply:
From training about how to write for ESL readers, we want to add the articles and participles, etc. ;-)